osf/models/node.py

```diff
@@ -2409,13 +2409,44 @@ def _add_related_source_tags(self, contributor):
         ).first() or osf_provider_tag
         contributor.add_system_tag(source_tag)
 
-    def _remove_from_associated_collections(self):
+    def _remove_from_associated_collections(self, auth=None, force=False):
```

Comment:
`force` makes it sound like we're forcing the removal from the collection, when really the question is whether this was invoked via Admin app, correct?

Reply:
Yeah I don't want to explicitly label it as for spam though because it could be some other type of admin action.